README.md

```diff
@@ -1 +1,31 @@
-# Dash-Seedling
+# 🌱 Dash-Seedling
+
+Template repository for building [dash](https://plotly.com/dash/) apps on [FlowEHR](https://github.com/UCLH-Foundry/FlowEHR).
```

Comment:
should you use `https://dash.plotly.com/` (which is the free version) rather than `https://plotly.com/dash/` (which references the enterprise version)?

Reply:
absolutely – thanks